### PR TITLE
Updates to debian/examples config file

### PR DESCRIPTION
**Pull request by patricklodder**
- pulled in bitcoin's new config file, and
- filled it in with dogecoin values, and
- properly formatted linebreaks, and
- renamed it to dogecoin.conf.

### CODE REVIEW

File: contrib/debian/examples/dogecoin.conf [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+##  If you run multiple nodes on a LAN, there's no need for ##
+##  all of them to open lots of connections.  Instead       ##
+##  'connect' them all to one node that is port forwarded   ##
+##  and has lots of connections.                            ##
+##       Thanks goes to [Noodle] on Freenode.               ##
+##############################################################
+
+# Use as many addnode= settings as you like to connect to specific peers
+#addnode=69.164.218.197
+#addnode=10.0.0.2:8333
```
Comment (langerhans) [inline contrib/debian/examples/dogecoin.conf]: Maybe change these to :22556. And is that an actual bitcoin node there?

File: contrib/debian/examples/dogecoin.conf [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+# Alternatively use as many connect= settings as you like to connect ONLY
+# to specific peers
+#connect=69.164.218.197
+#connect=10.0.0.1:8333
```
Comment (langerhans) [inline contrib/debian/examples/dogecoin.conf]: This too^

**Comment by patricklodder** [comment]
rpc ports have been fixed

**Comment by langerhans** [comment]
Thanks :)

**Pull request merged**